api_app/connectors_manager/views.py

```diff
@@ -19,14 +20,21 @@ def list(self, request):
 class ConnectorActionViewSet(PluginActionViewSet):
     queryset = ConnectorReport.objects.all()
 
-    def get_object(self, job_id, connector_name) -> ConnectorReport:
+    def get_object(self, job_id, name) -> ConnectorReport:
         try:
             return self.queryset.get(
                 job_id=job_id,
-                connector_name=connector_name,
+                name=name,
             )
         except ConnectorReport.DoesNotExist:
             raise NotFound()
 
     def _post_kill(self, report):
         pass
+
+    def _start_retry(self, report: ConnectorReport):
+        connectors_to_execute = [report.connector_name]
+        connectors_controller.start_connectors(
```

Comment:
missing the `runtime_configuration = {report.analyzer_name: report.runtime_configuration}` as is there in the `_start_retry` of `AnalyzerActionViewSet`.